.github/dependabot.yml

```diff
@@ -18,6 +18,14 @@ updates:
     directory: "/cmd/otelcorecol"
     schedule:
       interval: "weekly"
+  - package-ecosystem: "gomod"
+    directory: "/consumer"
+    schedule:
+      interval: "weekly"
+  - package-ecosystem: "gomod"
+    directory: "/internal/testdata"
+    schedule:
+      interval: "weekly"
```

Comment:
This is a package that we don't want to expose.

Reply:
I don't believe this exposes any additional modules.  They are still internal, IIUC.  It does reduce the number of dependencies you pull in when you reference this package.

Reply:
I started working on moving required functionality into the consumer package, but it involves copying basically the entire testdata package into consumer, and didn't seem worth it.  The cleanest way would be just to duplicate the package inside consumer: https://github.com/open-telemetry/opentelemetry-collector/pull/5578